README.md

```diff
@@ -1,9 +1,24 @@
 # serilog-sinks-rabbitmq
-![Build status](https://github.com/ArieGato/serilog-sinks-rabbitmq/actions/workflows/ci-build.yml/badge.svg)
-<a href="https://www.nuget.org/packages/Serilog.Sinks.RabbitMQ/">
-  <img src="https://img.shields.io/nuget/v/Serilog.Sinks.RabbitMQ.svg" />
-</a>
-[![Gitter](https://badges.gitter.im/serilog-sinks-rabbitmq/serilog-sinks-rabbitmq.svg)](https://gitter.im/serilog-sinks-rabbitmq/Lobby)
+
+![License](https://img.shields.io/github/license/ArieGato/serilog-sinks-rabbitmq)
+
+[![codecov](https://codecov.io/gh/ArieGato/serilog-sinks-rabbitmq/graph/badge.svg?token=Ma2sUoqqb1)](https://codecov.io/gh/ArieGato/serilog-sinks-rabbitmq)
```

Comment:
@ArieGato this token should be changed to one from https://app.codecov.io/gh/ArieGato/serilog-sinks-rabbitmq settings.

Reply:
```suggestion
[![codecov](https://codecov.io/gh/ArieGato/serilog-sinks-rabbitmq/graph/badge.svg?token=4uZ4oaOQUE)](https://codecov.io/gh/ArieGato/serilog-sinks-rabbitmq)
```